api/admin/routes.py

```diff
@@ -680,13 +690,8 @@ def roles():
       summary: Return a mapping from MARC codes to contributor roles.
       description: |
         This end point returns a map of MARC Codes to contributor roles that are currently available with this system.
-      responses:
-        200:
-          description: List of MARC codes mapped to Contributor roles
-          content: 
-            application/json:
-              schema: MARCRollContributorsDict
-            example:
+
+        example:
```

Comment:
I can see it now but it's not picking up the MD formatting. Could roll it up into the `description` block and I think it would show up formatted?